File: python/src/questao_3/main.py
from typing import List


def countEvenNumbers(numbers:List[int]):
    count = 0
    for number in numbers:
        if number % 2 == 0:
            count+=1
    return count



def main():
    print("Digite 10 numeros inteiros: ")
    numbers = []
    while len(numbers) < 10:
        numbers.append(int(input()))
    
    print("Quantida de numeros pares:",countEvenNumbers(numbers))




if __name__ == '__main__':
    main()

File: java/src/questao_3/Main.java
package questao_3;

import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

public class Main {

    public static int countEvenNumbers(List<Integer> numbers) {
        int count = 0;
        for (int number : numbers) {
            if (number % 2 == 0) {
                count++;
            }
        }
        return count;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        List<Integer> numbers = new ArrayList<>();
        System.out.println("Digite 10 números inteiros: ");
        while (numbers.size() < 10) {
            numbers.add(scanner.nextInt());
        }
        scanner.close();
        
        System.out.println("Quantidade de números pares: " + countEvenNumbers(numbers));
    }
}

